src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -898,6 +900,11 @@ bool goto_analyzer_parse_optionst::process_goto_program(
     remove_function_pointers(
       ui_message_handler, goto_model, options.get_bool_option("pointer-check"));
 
+    mm_io(goto_model);
+
+    // instrument library preconditions
+    instrument_preconditions(goto_model);
```

Comment:
Just noting that this isn't noted in the commit message. On a second thought, this change should probably go with the earlier commit that enabled library support in goto-analyzer.

Reply:
Fixed.